shell/probe/install.sh

```diff
@@ -49,7 +54,12 @@ done
 register_new_endpoint() {
     echo "[+] Provisioning Detect Endpoint Token..."
     local _token_url="${PRELUDE_API}/detect/endpoint"
-    ENDPOINT_TOKEN=$(curl -sfS -X POST -H "account:${PRELUDE_ACCOUNT_ID}" -H "token:${PRELUDE_ACCOUNT_SECRET}" -H "Content-Type: application/json" -d "{\"id\":\"${ENDPOINT_ID}\",\"tag\":\"darwin\"}"  "${_token_url}")
+    if [[ $ENDPOINT_TAGS ]];
+    then
+      local _tags=",\"tags\":$(echo "[\"$ENDPOINT_TAGS\"]" | sed 's/,/\",\"/g')"
```

Comment:
this seems super dicey. can't we just take the string of comma-separated tags?

Reply:
see original PR description: 

> Our backend currently only accepts a list, not a comma-separated list - if you're waiting for that update, it's coming, just lmk here!

Reply:
;) yes please

Reply:
hmm; @makk94 are you going to add the BE support?